Makefile

```diff
@@ -32,3 +33,60 @@ install:
 ## Start the geodesic shell by calling wrapper script
 run:
 	$(CLUSTER)
+
+## Target for testing cluster deployment
+test/gke/deploy: export CLOUDSDK_CONTAINER_CLUSTER = deepcell-test-$(shell bash -c 'echo $$RANDOM')
+test/gke/deploy:
+	# Some debug info
+	echo "TEST"
+	printenv
+	echo $(CLOUDSDK_CORE_PROJECT) && echo $(HOME)
+	pwd
+	ls
+	make init
+	# Installations of binaries
+	## helmfile
+	@wget https://github.com/roboll/helmfile/releases/download/v0.82.0/helmfile_linux_amd64
```

Comment:
Let's clean up this file with fewer debugging echos.

Also, I think we can move some of the "set up" functionality, like installing helm, to the `before_install` portion of travis.

Reply:
I agree on both points. Done.